packages/spectacle/package.json

```diff
@@ -7,10 +7,7 @@
   "files": [
     "lib/",
     "es/",
-    "dist/",
-    "examples/",
```

Comment:
Note that in the past we published `examples` so that it could be used upstream as a dep of `spectacle-ci` for generation. We don't have to continue doing this, but just for some historical context.

Reply:
That's good context! Hopefully with the monorepo setup, we won't have to ship those. (But also, the examples are no longer under the spectacle package, so I originally removed this because they wouldn't actually be included in the bundle as it currently stands.)